game-core/build.gradle

```diff
@@ -60,98 +46,17 @@ dependencies {
-            'icons/triplea_icon_48_48.png',
-            'icons/triplea_icon_64_64.png',
-            'icons/triplea_icon_128_128.png',
-            'icons/triplea_icon_256_256.png',
-            'install4j/macosx-amd64-1.8.0_144.tar.gz',
-            'install4j/windows-amd64-1.8.0_144.tar.gz',
-            'install4j/windows-x86-1.8.0_144.tar.gz'
-        ].each { path ->
-            download {
-                src "https://raw.githubusercontent.com/triplea-game/assets/master/$path"
-                dest "$buildDir/assets/$path"
-                overwrite false
-            }
-        }
-    }
-}
-
-import com.install4j.gradle.Install4jTask
-task generateInstallers(type: Install4jTask, dependsOn: [shadowJar, downloadAssets], group: 'release') {
-    projectFile = file('build.install4j')
-    release project.version
-    doFirst {
-        logger.lifecycle("building installer release of version '${project.version}'")
-    }
-}
-
-task generateInstallerReleases(group: 'release', dependsOn: [generateInstallers]) {
-    doLast {
-        ant.chmod(dir: releasesDir, perm: '+x', includes: '*.sh')
-    }
-}
-
-task generateZipReleases(type: Zip, group: 'release', dependsOn: shadowJar) {
-    baseName = 'triplea'
-    classifier = 'all_platforms'
-    ['assets', 'dice_servers'].each { folder ->
-        from(folder) {
-            into(folder)
-        }
-    }
-    from(file('game_engine.properties'))
-    from(shadowJar.outputs) {
-        into('bin')
-    }
-}
-
-task release(group: 'release', dependsOn: [generateZipReleases, generateInstallerReleases]) {
-    doLast {
-        publishArtifacts([
-            file("$distsDir/triplea-${version}-all_platforms.zip"),
-            file("$releasesDir/TripleA_${version}_macos.dmg"),
-            file("$releasesDir/TripleA_${version}_unix.sh"),
-            file("$releasesDir/TripleA_${version}_windows-32bit.exe"),
-            file("$releasesDir/TripleA_${version}_windows-64bit.exe")
-        ])
-    }
-}
-
-gradle.taskGraph.whenReady { graph ->
-    graph.getAllTasks().any({
-        if (it.name == "generateInstallers") {
-            if (!project.hasProperty('install4jHomeDir')) {
-                File propertiesFile = file("${System.getProperty('user.home')}/.gradle/gradle.properties")
-                throw new RuntimeException("Specify install4jHomeDir in $propertiesFile")
-            }
-            def p = file(project.install4jHomeDir)
-            logger.lifecycle('using install4j home directory ' + p.getAbsolutePath())
-            it.project.install4j.installDir = file(project.install4jHomeDir)
-        }
-    })
-}
-
 checkstyleMain {
     maxWarnings = checkstyleMainMaxWarnings.toInteger()
 }
 
 checkstyleTest {
     maxWarnings = checkstyleTestMaxWarnings.toInteger()
 }
+
+task cleanRemoteLibs(
+        type: Delete,
+        group: LifecycleBasePlugin.BUILD_GROUP,
+        description: 'Deletes the remote libraries directory.') {
+    delete remoteLibsDir
```

Comment:
A second thought:
The substance libraries should be available using jitpack now, I convinced the author to update his ant scripts, so jitpack can generate jars from it.
This has the downside that we won't have a clear distinction between substance and trident anymore, but we'd have a reliable maven-like repository and could drop the custom code.

Reply:
Agreed.  I was going to extract this stuff to an external script (thinking it might have to be re-used across multiple projects).  I'll investigate using Jitpack instead.

I've also been following kirill-grouchnikov/radiance#6, which will finally deploy the artifacts to Maven repos.

Reply:
Great to hear that👍